Types/InsertChangeSet.php

```diff
@@ -66,21 +68,88 @@ public static function columns(array $tokens) {
      * @SuppressWarnings("PHPMD.StaticAccess")
      */
     public static function values(array $tokens) {
-        $values = explode(',', self::removeBrackets(self::removeSpacesBetweenComma(end($tokens['VALUES'])['base_expr'])));
+        $values = self::removeBrackets(end($tokens['VALUES'])['base_expr']);
 
         return array_map(function($value) {
             return Value::create($value);
-        }, $values);
+        }, static::parseValueStringToArray($values));
     }
 
     /**
-     * removes spaces between commas
+     * parses the values string into a an array that has the correct amount of values, compared with the column array.
```

Comment:
`Parse the string values into an array that has the correct amount of values, compared with the column array`

Reply:
done